tests/test_dockerflow.py

```diff
@@ -4,7 +4,7 @@
 
 import json
 
-from tests.utils import phab_url, trans_url
+from tests.utils import phab_url
```

Comment:
Can we file a bug for removing `trans_url` and other transplant related test code?

Reply:
Sure, most of the tansplant related test code was removed in [bug 1654646](https://bugzilla.mozilla.org/show_bug.cgi?id=1654646) but there are some stragglers left like this one. Filed [bug 1831622](https://bugzilla.mozilla.org/show_bug.cgi?id=1831622).